jax/_src/lax/lax.py

```diff
@@ -5898,21 +5898,27 @@ def _stop_gradient_batch_rule(batched_args, batch_dims):
 batching.primitive_batchers[ad_util.stop_gradient_p] = _stop_gradient_batch_rule
 
 
-def create_token(x):
+def create_token(x=None):
   """Creates an XLA token value with no preconditions for sequencing effects.
 
   Experimental.
 
   Args:
-    x: a dummy argument used to tie the CreateToken operator into a trace. The
-       value of `x` is ignored.
+    x: Optional dummy argument used to tie the CreateToken operator into a
```

Comment:
Let's be even more bold here: just say the argument is ignored and nothing else. Consider the docstring of `lax.tie_in` itself!

WDYT about a docstring of:

```python
def create_token(_=None):
  """Creates an XLA token value with no preconditions for sequencing effects.

  Experimental.

  The argument is ignored. It exists for backward compatibility.
  """
```

Reply:
Good call.